src/registrar/models/host_ip.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.core.validators import validate_ipv46_address
+
+from .utility.time_stamped_model import TimeStampedModel
+from .host import Host
+
+
+class HostIP(TimeStampedModel):
+    """
+    Hosts may have one or more IP addresses.
+
+    The registry is the source of truth for this data.
+
+    This model exists ONLY to allow a new registrant to draft DNS entries
+    before their application is approved.
+    """
+
+    address = models.CharField(
+        max_length=46,
+        null=False,
+        blank=False,
+        default=None,  # prevent saving without a value
+        validators=[validate_ipv46_address],
+        help_text="IP address",
+    )
+
+    host = models.ForeignKey(
+        Host,
+        on_delete=models.PROTECT,
```

Comment:
**question (non-blocking)**: This seems maximally safe to force us to manually clean up a host's IP addresses before we delete the host, but I wonder if it might be reasonable to just `models.CASCADE` here and auto-delete these IP addresses when we delete the host?

Reply:
I'll leave it as `models.PROTECT` for the moment, because I believe this mirrors what EPP protocol specifies for the registry. (IP addys must be deleted before their hosts can be.) However, since this model is only a scratchpad for draft data that has not been published to the registry yet, I think we could absolutely change it later, if it ends up being more useful as `models.CASCADE`.